resources/datadog-integrations/validate_manifest.rb

```diff
@@ -21,6 +21,22 @@ def validate_manifest_0_1_0(manifest_hash)
   validate_manifest_loop(manifest_hash, mandatory_fields)
 end
 
+def validate_manifest_0_1_1(manifest_hash)
+  mandatory_fields = [
+    "maintainer",
+    "manifest_version",
+    "max_agent_version",
+    "min_agent_version",
+    "name",
+    "short_description",
+    "support",
+    "type",
+    "version",
+    "guid",
+  ]
+  validate_manifest_loop(manifest_hash, mandatory_fields)
+end
+
```

Comment:
I would guess the mandatory fields for manifest 1.0.0 below would be a superset of the mandatory fields for 0.1.1?